smallrye-reactive-messaging-kafka/src/test/java/io/smallrye/reactive/messaging/kafka/tracing/TracingPropagationTest.java

```diff
@@ -59,15 +59,22 @@
     @BeforeAll
     static void setupOpenTelemetry() {
         // We need to set a propagator
-        OpenTelemetry.setPropagators(
-                DefaultContextPropagators.builder().addTextMapPropagator(HttpTraceContext.getInstance()).build());
+        OpenTelemetry otelInstance = OpenTelemetrySdk.builder().setPropagators(
+                ContextPropagators.create(W3CTraceContextPropagator.getInstance()))
+                .build();
+
+        GlobalOpenTelemetry.set(otelInstance);
     }
 
     @BeforeEach
     public void setup() {
         testExporter = InMemorySpanExporter.create();
-        spanProcessor = SimpleSpanProcessor.newBuilder(testExporter).build();
-        OpenTelemetrySdk.getTracerManagement().addSpanProcessor(spanProcessor);
+        spanProcessor = SimpleSpanProcessor.builder(testExporter).setExportOnlySampled(false).build();
+        OpenTelemetrySdk.getGlobalTracerManagement().addSpanProcessor(spanProcessor);
+
+        // Without this, TRACER can be set to a tracer prior to the above configuration being active.
+        // Resulting in no traces being captured
+        KafkaConnector.TRACER = GlobalOpenTelemetry.getTracerProvider().get("io.smallrye.reactive.messaging.kafka");
```

Comment:
Hum, I see. Not great...

Reply:
Tried a slightly different fix that retains static, currently running build now

Reply:
Nothing else I try can "setup" the bits in OpenTelemetry early enough such that the static in `KafkaConnector` can "do the right thing"